docs/src/css/app.sass

```diff
@@ -167,3 +231,27 @@ body.q-scroll--lock .q-page
 
   > .q-scroll--anchor
     overflow-anchor: auto
+
+::-webkit-scrollbar 
```

Comment:
Can you add a comment explaining what these fixes do and why are they needed?

Reply:
Ok. 

Scott